daemon/daemon.go

```diff
@@ -476,6 +479,8 @@ func (d *daemon) createEventFromEventFile(ef store.EventFile) error {
 	return d.createEvent(ev)
 }
 
+
+//todo: DOES NOT CREATE EVENT, IT READS CONFIGURATION FILE FROM HOST AND CALL CREATEEVENT FUNCITON WITHHIN IT
```

Comment:
So then it *does* create an event from a config file, doesn't it?

Reply:
It were but I have modified it to send some stream of data back to client.